app/daemon.go

```diff
@@ -3,9 +3,8 @@ package app
 import (
 	"fmt"
 	"net/http"
-	"os"
-
 	_ "net/http/pprof" // for runtime profiling
+	"os"
```

Comment:
Any reason we are moving this here?

Reply:
just the default formatter by goimports, nothing special.